tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, py33, py34, py35, py36, py37, lint, lint-py2, lint-py3
+envlist = py37, py38, py39, lint
```

Comment:
(optional) We don't test for python 3.10?

Reply:
That's not yet available on our Jenkins. I'll ask Dorian if he can install it :)

Reply:
Ah I see, I think it's fine like this either way 🙈

Reply:
Yeah, I think it's more important to test older versions as the new ones should be backward compatible, but why not test it if possible :)